content/blog/2016/2016-07-04-ewm-alpha-version.adoc

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "GSoC: External Workspace Manager Plugin alpha version"
+tags:
+- pipeline
+- plugins
+- gsoc
+author: alexsomai
+---
+
+[NOTE]
+--
+This is a guest post by link:https://github.com/alexsomai[Alexandru Somai], Google Summer of Code student
+at Jenkins project.
+--
+
+An alpha version of the External Manager Plugin has been released!
+To be able to install this plugin, you must follow the steps from the Experimental Plugins Update Center
```

Comment:
It would be great if you add a pre-history and a short plugin description directly to the post